cos_registration_agent/tls_utils.py

```diff
@@ -0,0 +1,28 @@
+"""Module to handle device TLS certificates."""
+
+import logging
+import os
+
+from cos_registration_agent.write_data import write_data
+
+logger = logging.getLogger(__name__)
+
+
+def save_device_tls_certs(cert: str, key: str, certs_dir: str) -> None:
+    """Extract TLS cert/key from HTTP response JSON and save to folder.
+
+    Args:
+        response: HTTP response containing 'certificate' and 'private_key'
```

Comment:
This docstring seems outdated.

Reply:
Fixed in [ab714d1](https://github.com/canonical/cos-registration-agent/pull/53/commits/ab714d14c07e5f357697e4f86d647f675d2fa9b2)